modules/serving-transport-encryption-trust-configuration.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * knative-serving/serving-transport-encryption.adoc
+:_mod-docs-content-type: CONCEPT
+[id="serving-transport-encryption-trust-configuration_{context}"]
+= Trust configuration
+
+When you enable any of the transport encryption features, you must make sure that all clients calling do trust the Certificate Authority (CA) issuing the certificates used for the transport encryption.
```

Comment:
Is this the same as the "callers" mentioned earlier?
```suggestion
When you enable any of the transport encryption features, you must make sure that all callers trust the Certificate Authority (CA) issuing the certificates used for the transport encryption.
```

if not, still has an extra "do":
```suggestion
When you enable any of the transport encryption features, you must make sure that all clients calling trust the Certificate Authority (CA) issuing the certificates used for the transport encryption.
```